numpy/core/numeric.pyi

```diff
@@ -47,35 +106,38 @@ def full_like(
     subok: bool = ...,
     shape: Optional[_ShapeLike] = ...,
 ) -> ndarray: ...
+@overload
+def count_nonzero(
+    a: ArrayLike, axis: None = ..., *, keepdims: Literal[False] = ...
+) -> int: ...
```

Comment:
A bit odd that one overload returns a builtin `int` and the other one a numpy `signedinteger` (or an array).
See https://github.com/numpy/numpy/issues/17562.